app/(event)/live/_components/schedule.tsx

```diff
@@ -144,13 +147,31 @@ const ScheduleSection = () => {
             <p className="text-xs ">Last refreshed {timeAgo}</p>
           </div>
           <ScheduleFilters activeFilters={activeFilters} setActiveFilters={setActiveFilters} />
+          <div className="flex items-center space-x-2 pt-1">
+            <Switch
+              id="show-previous-events"
+              checked={showPrevEvents}
+              onCheckedChange={() => setShowPrevEvents(!showPrevEvents)}
+            />
+            <Label htmlFor="show-previous-events">Show Previous Events</Label>
```

Comment:
```suggestion
            <Label htmlFor="show-previous-events">Show Past Events</Label>
```
I think this wording should be "Show Past Events" instead. Let me know if you disagree.